plugins/line-numbers/prism-line-numbers.js

```diff
@@ -102,13 +109,37 @@
 		lineNumbersWrapper.className = 'line-numbers-rows';
 		lineNumbersWrapper.innerHTML = lines;
 
-		if (pre.hasAttribute('data-start')) {
-			pre.style.counterReset = 'linenumber ' + (parseInt(pre.getAttribute('data-start'), 10) - 1);
+		if (pre.hasAttribute('data-line-numbers-start')) {
```

Comment:
I understand why you want to rename this data attribute... but it looks quite BC-breaking to me. We would need to update the docs, etc.

I would suggest you do the exact opposite: rename the data attribute used in line-highlight. AFAIK it's only used for the CSS ;)